lib/tasks/data_load.rake

```diff
@@ -174,4 +174,27 @@ namespace :data do
     desc "Snapshot, copy, and import local database to Heroku"
     task :import_snapshot => [:setup, :snapshot_local, :copy_snapshot_to_s3, :export]
   end
+
+  task validate: :environment do
+    tables = ActiveRecord::Base.connection.tables;nil
+
+    tables.each do |table|
+      begin
+        z = table.singularize.camelize.constantize
+        records = z.all;nil
+
```

Comment:
I believe ```Table.all``` will load everything into memory.   ```Table.find_each do |record|``` loads 1,000 at a time - may be the way to go in the event of large tables.